### ISSUE TITLE
Error while request tags: TaggerRequestHandler requires text to be POSTed to it

### ISSUE THREAD

**Issue by squallsama** [OpenSextant/SolrTextTagger#77]
Even if I use QueryRequest with METHOD.POST I need to use deprecated override:
@Override
        public Collection<ContentStream> getContentStreams() {
            return Collections.singleton((ContentStream) new ContentStreamBase
                    .StringStream(input));
        }

To get success with solr text tagger request handler.

**Comment by dsmiley** [comment]
Yes, it's fairly recent that getContentStreams was deprecated in favor of `org.apache.solr.client.solrj.SolrRequest#getContentWriter`.  But it'll be around thru the 7.x series for back-compat reasons.  You could send the stream.body param as an alternative.
